### PR TITLE
Fixed TabScrollButtons alignment and styles

### LINKED ISSUE #1543: The tabview scroll buttons aren't properly aligned
**Describe the bug**
The scroll buttons in tabview aren't properly aligned.

**To Reproduce**
1. Launch files-uwp app.
2. Go to a folder with sufficiently large path.
3. Right click on it and open it in new tab.
4. Open the vertical tabview flyout.

**Expected behavior**
The scroll buttons should be centered.

**Screenshots**
ApplicationFrameHost_v7vPqEoW7p

**Desktop (please complete the following information):**
 - OS Version : Windows 10 1903 18362.239
 - App version: v0.12.0

**Additional context**
Also the the style of button should match the style of rest of buttons in the app.

### PR DESCRIPTION
**Pull request by soumyamahunt**
Fixes #1543.
| Before      |         After |
| :--------------------------------- | :-------------------- |
| ApplicationFrameHost_v7vPqEoW7p | ApplicationFrameHost_4GnGAxIuh6 |

### CODE REVIEW

**Comment by soumyamahunt** [comment]
> Each item should only of 12px margin.

You mean the Tabitems??
 > Don't add buttons to go left or right.

Which buttons?? I didn't add any button, I just modified the scroll buttons' styles.

**Comment by Jaiganeshkumaran** [comment]
> > Each item should only of 12px margin.
> 
> You mean the Tabitems??
> 
> > Don't add buttons to go left or right.
> 
> Which buttons?? I didn't add any button, I just modified the scroll buttons' styles

Don't modify scroll button styles or anything just fit each tab item full container. Currently there's a lot of space wasted.

**Comment by soumyamahunt** [comment]
> Don't modify scroll button styles or anything just fit each tab item full container. Currently there's a lot of space wasted.

Users need these buttons to distinguish between longer paths and see the full path before the tab is loaded, once tab is loaded the path text will be replaced by the title so the buttons would disappear. I don't see any waste there.

**Comment by Jaiganeshkumaran** [comment]
@soumyamahunt If you look closely there are a lot of space left and right. They should be reduced and the tab items should be stretched. The scroll button's width should be decresed and margin should be reduced. Also currently only the parent folders path is shown rather than the full path which makes it confusing.

**Comment by soumyamahunt** [comment]
> @soumyamahunt If you look closely there are a lot of space left and right. They should be reduced and the tab items should be stretched. The scroll button's width should be decresed and margin should be reduced. 

I removed the horizontal scrolling entirely and instead opted for tooltips.
> Also currently only the parent folders path is shown rather than the full path which makes it confusing.

I have filed an issue report about it. Will address in my next pr.

**Comment by XPoppyX** [comment]
What if the size of the tab is limited? that the text is trimmed, since anyway you can see the full text of the header in tooltip

**Comment by tsvietOK** [comment]
I like what @XPoppyX proposed. But in tooltip we could display full path to the folder. What do you think about it?

**Comment by soumyamahunt** [comment]
@XPoppyX @tsvietOK updated design.
ApplicationFrameHost_k4alBCc8Zk (2)

File: Files/UserControls/VerticalTabView.xaml [hunk 1]
```diff
@@ -430,6 +437,137 @@
             </Setter>
         </Style>
 
+        <Style x:Key="DefaultTabViewListViewStyle" TargetType="primitives:TabViewListView">
+            <Setter Property="VerticalAlignment" Value="Top" />
```
Comment (duke7553) [inline Files/UserControls/VerticalTabView.xaml:441]: Why was it necessary to modify this Style?

Comment (soumyamahunt) [reply Files/UserControls/VerticalTabView.xaml]: Yeah, we don't need that, I have removed it.

**Pull request merged**